### PR TITLE
Allow Service Annotations on hm-basic-webapp

### PR DESCRIPTION
**Pull request by scudelletti**

### CODE REVIEW

File: hm-basic-webapp/templates/service.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ kind: Service
 metadata:
   name: {{ include "hm-basic-webapp.prefix" . }}
   labels: {{ toYaml .Values.serviceLabels | nindent 8 }}
+  annotations: {{ toYaml .Values.serviceAnnotations | nindent 8 }}
```
Comment (scudelletti) [inline hm-basic-webapp/templates/service.yaml]: I think they should be 4 not 8. @slashmili WDYT?

```
# Source: sample-app/charts/hm-basic-webapp/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: develop-sample-app
  labels:
        foo: bar
        service-discovery-name: sample-app
  annotations:
        foo: bar
        some_key: some_value
spec:
  ports:
    - name: http
      port: 80
      targetPort: 9292
  selector:
    app: develop-sample-app
  type: ClusterIP
```

Comment (slashmili) [reply hm-basic-webapp/templates/service.yaml]: yup, the rest of the yaml has 2 space tabs. so 4 is correct. I think I copied label line from somewhere else.

**Review by slashmili — APPROVED** [review]
👍🏼

File: hm-basic-webapp/templates/service.yaml (hunk 1 above)
Comment (scudelletti) [reply hm-basic-webapp/templates/service.yaml]: ✨ After the change from 4 to 8:

```
# Source: sample-app/charts/hm-basic-webapp/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: develop-sample-app
  labels: 
    foo: bar
    service-discovery-name: sample-app
  annotations: 
    foo: bar
    some_key: some_value
spec:
  ports:
    - name: http
      port: 80
      targetPort: 9292
  selector:
    app: develop-sample-app
  type: ClusterIP
  ```

**Pull request merged**